src/lib/uTop.ml

```diff
@@ -15,6 +15,8 @@ open LTerm_text
 open LTerm_geom
 open LTerm_style
 
+module M = Directories.Base_dirs ()
```

Comment:
I don't think that's the best way to do it, it would probably be better like this:

```ml
module M = Directories.Project_dirs (struct
    let qualifier = "org"
    let organization = "ocaml-community"
    let application = "utop"
end)
```
And you'll have to create the directory yourself if it doesn't exist.

Reply:
Alright, I will add that, I was wondering if this was how I had to do it but I didn't know what to put there.